buffalo/cmd/dev.go

```diff
@@ -68,8 +68,8 @@ This behavior can be changed in your .buffalo.dev.yml file.`,
 
 func startWebpack(ctx context.Context) error {
 	app := meta.New(".")
-	if !app.WithWebpack {
-		// there's no webpack, so don't do anything
+	if !app.WithWebpack && !app.WithYarn {
+		// there's no webpack or yarn, so don't do anything
```

Comment:
Do you need this check? If there’s no webpack config is there yarn? Or rather what’s the situation where there’s yarn, but no webpack?

Reply:
If you want to use a yarn script, you don't necessarily need to use webpack. With the template that is supplied as part of this PR we can use either webpack or yarn but it would be very easy for someone to modify the yarn script to do something else - such as building the assets directly.